src/LibraryManager.Vsix/UI/Controls/PackageContentsTreeViewAutomationPeer.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System.Windows.Automation.Peers;
+using System.Windows.Controls;
+using Microsoft.Web.LibraryManager.Vsix.Resources;
+
+namespace Microsoft.Web.LibraryManager.Vsix.UI.Controls
+{
+    /// <summary>
+    /// Custom AutomationPeer for PackageContentsTreeView control
+    /// </summary>
+    internal class PackageContentsTreeViewAutomationPeer : UserControlAutomationPeer
+    {
+        public PackageContentsTreeViewAutomationPeer(UserControl owner) : base(owner)
+        {
+        }
+
+        protected override string GetLocalizedControlTypeCore()
```

Comment:
>GetLocalizedControlTypeCore [](start = 34, length = 27)

See other comment on getLocalizedControlTypeCore

Reply:
Updated in iteration 4.

---
In reply to: [210109867](https://github.com/aspnet/LibraryManager/pull/346#discussion_r210109867) [](ancestors = 210109867)